src/string_bytes.cc

```diff
@@ -305,10 +305,10 @@ size_t StringBytes::Write(Isolate* isolate,
               input_view.length());
         }
       } else {
-        String::Value value(isolate, str);
+        String::ValueView value(isolate, str);
```

Comment:
You could re-use `input_view` here to avoid duplicate variables, ditto below (doesn't change much except making the code more obviously correct and maybe allowing the compiler to remove an extra assertion or two)

Reply:
Got it. I'll update this now

(not sure when/how I removed the https://github.com/nodejs/node/labels/needs-ci label :thinking:)